packages/n/nano/package.yml

```diff
@@ -11,9 +11,9 @@ description: |
     GNU nano is an easy-to-use text editor originally designed as a replacement for Pico, the ncurses-based editor from the non-free mailer package Pine (itself now available under the Apache License as Alpine).
 mancompress: yes
 builddeps  :
-    - nanorc
+    - nano-syntax-highlighting
 rundeps    :
-    - nanorc
+    - nano-syntax-highlighting
```

Comment:
Have you tested an upgrade scenario? My gut feeling here is that it will cause an error since nanorc will still be installed but the updated nano will pull in nano-syntax-highlighting which will cause a conflict.

Reply:
Yes, I used the go-task local to test this. Had nanorc in my local repo.